src/components/Products/ProductsListItem.vue

```diff
@@ -1,111 +1,81 @@
 <template>
-  <div class="product-list-item">
-    <div
-        class="product-list-item__picture"
-        :style="{ backgroundImage: 'url(' + imgUrl + ')' }"
-    ></div>
-    <div class="product-list-item__description">
-      <span
-          class="product-list-item__description__title"
-          v-if="name.length < 20"
-      >{{ name }}
+	<div class="product-list-item">
+		<div class="product-list-item__picture"
+         :style="{ backgroundImage: 'url(' + imgUrl + ')' }"></div>
+		<div class="product-list-item__description">
+			<span class="product-list-item__description__title">
+        {{ name }}
       </span>
-			<span class="product-list-item__description__title" v-else>
-        {{ name.substring(0, 14).concat("..") }}
-      </span>
-      <span class="product-list-item__description__price">${{ price }}</span>
-    </div>
-    <div class="product-list-item__rating">
-      <IconStar class="product-list-item__star"/>
-      <span>{{ rating }} </span>
-    </div>
-  </div>
+			<span class="product-list-item__description__price">${{ price }}</span>
+		</div>
+		<div class="product-list-item__rating">
+			<IconStar class="product-list-item__star" />
+			<span>{{ rating }} </span>
+		</div>
+	</div>
 </template>
 
 <script lang="ts">
-import {mapGetters} from "vuex";
+import { mapGetters } from "vuex";
 import IconStar from "@/assets/icons/icon-stars.vue";
 
 export default {
-  components: {
-    IconStar,
-  },
-  name: "ProductsListItem",
-  props: {
-    id: String,
-    name: String,
-    type: String,
-    price: Number,
-    imgUrl: String,
-    rating: Number
-  },
-  computed: {
-    ...mapGetters("cart", ["cartItems"])
-  }
+	components: {
+		IconStar
+	},
+	name: "ProductsListItem",
+	props: {
+		id: String,
+		name: String,
+		type: String,
+		price: Number,
+		imgUrl: String,
+		rating: Number
+	},
+	computed: {
```

Comment:
you should move `computed` to `setup` function which is part of `vue-composition-api` feature,
also if you want to use something like `mapGetters` you can check `vue-composition-helpers`
https://github.com/greenpress/vuex-composition-helpers